src/server/contentSecurityPolicy.ts

```diff
@@ -165,6 +165,8 @@ const frameSrc = (() => {
     '*.worldbank.org',
     'embed.molview.org',
     'embed.ted.com',
+    'reader.pubfront.com',
```

Comment:
Er det en grunn til at denne også fulgte med?

Reply:
Her skrev jeg før jeg leste kommentaren din. Fungerer det altså ikke uten dette domenet?

Reply:
Det fungerte ikke for meg uten å ha med det domenet